misc-utils/waitpid.c

```diff
@@ -0,0 +1,202 @@
+/*
+ * waitpid(1) - wait for process termination
+ *
+ * Copyright (C) 2022 Thomas Weißschuh <thomas@t-8ch.de>
+ *
+ * This program is free software; you can redistribute it and/or modify
```

Comment:
What about replacing the license blurbs with SPDX?

Reply:
Yes, please; I already used SPDX in some places, and for the new code, it should be a "must-have".